src/coreclr/vm/gcinfodecoder.cpp

```diff
@@ -786,7 +786,7 @@ bool GcInfoDecoder::EnumerateLiveSlots(
 
                 UINT32 normStart = lastNormStop + normStartDelta;
                 UINT32 normStop = normStart + normStopDelta;
-                if(normBreakOffset >= normStart && normBreakOffset < normStop)
+                if(normBreakOffset >= normStart && normBreakOffset <= normStop)
```

Comment:
This is an ugly hack to workaround a rare case where a call to a hijacked method is immediately followed by a NoGC region thus the caller IP is outside of interruptible code. 
It is always safe to pop to the caller of a regular managed method and start stackwalk at the call site, but when we do that as a part of hijack and the above condition present, it triggers asserts if caller is also fully-interruptible.

A proper fix for this condition is in PR: https://github.com/dotnet/runtime/pull/103540